### PR TITLE
Cleanup misc to reflect kubefed rename

### PR DESCRIPTION
**Pull request by marun**
Partial fix for #936

### CODE REVIEW

File: charts/kubefed/charts/controllermanager/templates/crds.yaml [hunk 1]
```diff
@@ -579,9 +579,9 @@ spec:
               - resourceLock
               type: object
             scope:
-              description: The scope of the kubefed control plane should be either
-                `Namespaced` or `Cluster`. `Namespaced` indicates that the kubefed
-                namespace will be the only target for federation.
+              description: The scope of the KubeFed control plane should be either
+                `Namespaced` or `Cluster`.  `Namespaced` indicates that the KubeFed
```
Comment (pmorie) [inline charts/kubefed/charts/controllermanager/templates/crds.yaml]: nit: i suspect @abrennan89 would have an opinion on two spaces vs. one here, but not a reason to hold the PR

**Comment by pmorie** [comment]
This moves the ball forward I'm sure we'll continually comb over docs so I'd prefer to get this swath merged.

/lgtm

File: charts/kubefed/charts/controllermanager/templates/crds.yaml (hunk 1 above)
Comment (marun) [reply charts/kubefed/charts/controllermanager/templates/crds.yaml]: I'll fix in a follow-up.

Comment (marun) [reply charts/kubefed/charts/controllermanager/templates/crds.yaml]: See #942

**Pull request merged**